misc/sparse_matrix.py

```diff
@@ -0,0 +1,64 @@
+import taichi as ti
```

Comment:
Actually, in `misc/sparse_matrix.py`, you can simply use
```
print(">>> K.print_triplets()")
...
print(">>> A = K.build()")
...
print(">>> C = A + A")
```
instead of 
```
print(">>>>>>>> Before build: ")
...
print(">>>>>>>> After build: ")
...
print(">>>> Summation test")
```
as the hint strings.

The former is simpler and more informative.

Reply:
Good point. Thank you!